samples/custom-login/public/index.html

```diff
@@ -0,0 +1,192 @@
+<html>
+  <head>
+    <!-- https://global.oktacdn.com/okta-auth-js/4.0.0/okta-auth-js.polyfill.js -->
+    <script src="/okta-auth-js.polyfill.js" type="text/javascript"></script>
+
+    <!-- https://global.oktacdn.com/okta-auth-js/4.0.0/okta-auth-js.min.js -->
+    <script src="/okta-auth-js.min.js" type="text/javascript"></script>
```

Comment:
Wait, our polyfill bundle is ONLY the polyfill?   I was under the impression it was authJS+polyfill.  I'll have to check the README to see if I'm just wrong or if other devs will have this impression.

Reply:
Samples are now generated from templates, so the version will update automatically based on what is installed in node_modules

Reply:
Yes the polyfill can also be used alongside the SIW (which implicitly includes authJS) to support IE 11.

Reply:
The signin widget does have a bundle which includes a polyfill and one without. However I think the SIW "polyfill" does not make global changes, it is a babel polyfill for ES features, and there may no longer be a great reason to use the SIW bundle without polyfill unless you were re-babeling it and running into a conflict. I'm not sure that the SIW polyfill actually provides compatibility for IE 11. All the stuff that breaks in IE11 happens in authJS. I'll have to take a closer look at this at some point.